dev/doc/xml-protocol.md

```diff
@@ -753,3 +754,41 @@ Ex: `status = "Idle"` or `status = "proof: myLemmaName"` or `status = "Dead"`
 </feedback>
 ```
 
+## <a name="highlighting">Highlighting</a>
+
+Proof diffs, which are enabled by the `Diffs` option, highlight differences
+between the current and previous proof states in the displayed output.
+These are represented in output fields in the XML protocol with embedded
+XML-like tags.
```

Comment:
Why "XML-like tags" and not just "XML tags"? AFAIK XML allows defining whatever tags you want.

Reply:
An XML document has a particular tag structure/syntax.  IIRC, embedding diff tags into string *values* is not an official part of XML.  IIRC you have to use an alternate construct so the embedded tags are escaped and don't cause a parsing error.  I think I'll just use the term "tags".  The reader will see the form in the examples.

Reply:
FWIW, there are languages for specifying the structure of XML documents, and frequently the spec documents drive parsing.  But it seems unlikely these languages are supported in OCaml.

Reply:
> embedding diff tags into string _values_ is not an official part of XML

OK I get it.